PublicMapping/districtbuilder
# Use JSON configuration to control fields displayed in metrics table

## Overview

In the first of what we anticipate to be a number of such requests, Dane County has asked to support both "Other" and "Multiracial" as demographics, in addition to the demographics we already have for WI.

To support this, I've removed our hardcoded list of fields we support to instead be based on what is configured in the `static-metadata.json` configuration file we create when preprocessing data.

We still have a hard-coded list that we use for sorting though, instead of relying solely on the sort order in `static-metadata.json` (any fields not on the hard-coded list, like "Multiracial", will sort to the end of the grouping).

### Checklist


### Demo

Note the "Other" and "Multiracial" columns:
![Capture](https://user-images.githubusercontent.com/4432106/131183270-b3ef0057-0f12-477d-9333-fbf92112bbe2.PNG)
Pinning works for the new column:
![Capture2](https://user-images.githubusercontent.com/4432106/131183271-5ac92409-a6fd-4906-aa4b-3cb83efc84b7.PNG)
Existing region still working as expected:
![Capture3](https://user-images.githubusercontent.com/4432106/131183272-d05105b2-0ef5-4222-9f2a-0a51ec0fbc14.PNG)


## Testing Instructions

 - Using `scripts/dbshell`, run `TRUNCATE region_config CASCADE` to delete your regions / projects, then run `scripts/load/dev-data` to load new testing data
- Go to the testing organization http://localhost:3003/o/azavea and create a map for the Dane County template
- Use the double-arrow button in the sidebar to expand the metrics viewer
  - You should see columns for Native, Pacific, Other, and Multiracial
  - You should _not_ see a row for Multiracial in the map / sidebar tooltips

Closes #973
